bench/arrow/optimized/grouper.cc.ll

```diff
@@ -27073,7 +27073,7 @@ for.body.i:                                       ; preds = %for.cond.preheader.
   br i1 %exitcond52.not.i, label %if.end32.i, label %for.body.i, !llvm.loop !565
 
 if.else.i:                                        ; preds = %while.body.i
-  %cmp.i16.i = icmp eq i16 %block.sroa.5.0.extract.trunc.i, 0
+  %cmp.i16.i = icmp ult i32 %call.i, 65536
```

Comment:
Regression.

Reply:
Duplicate of https://github.com/dtcxzyw/llvm-opt-benchmark/pull/600#discussion_r1597405950